.azure_pipelines/job_templates/olive-build-template.yaml

```diff
@@ -40,14 +43,26 @@ jobs:
       versionSpec: 3.8
     displayName: Use Python 3.8
 
-  - script: make install-olive INSTALL_EXTRAS=[$(device)]
+  - script: pip install .
     displayName: Install Olive
 
+  - ${{ if startsWith(parameters.onnxruntime, 'ort-nightly') }}:
+    - script: |
+        pip install ${{ parameters.onnxruntime }} --index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/
```

Comment:
https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ 

Since we are installing with `--index-url`. The requirements for onnxruntime must be installed beforehand.